pkg/api/config/config.go

```diff
@@ -121,6 +121,13 @@ type SchedulerConfig struct {
 	NumWorkers int
 }
 
+// ClusterConfig is the scale-out configuration which is identical for all
+// replicas.
+type ClusterConfig struct {
+	Members []string `json:"members" mapstructure:"members"`
+	HashKey string   `json:"hashKey" mapstructure:"hashKey"`
```

Comment:
Add a comment that we use "SipHash" as the hashing algorithm, so we need a hash key.

Reply:
Added comments.